tdrs-backend/docker-compose.local.yml

```diff
@@ -29,6 +29,18 @@ services:
       # Copy in the Localstack setup script to configure any buckets needed
       - ../scripts/localstack-setup.sh:/docker-entrypoint-initaws.d/localstack-setup.sh
 
+  elastic:
+    image: elasticsearch:7.17.6
+    environment:
+      - discovery.type=single-node
+      - xpack.security.enabled=false
```

Comment:
Why is this set to false? Connection issues?

Found [this](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-settings.html#general-security-settings) for posterity.

Reply:
this setting will be `true` in deployed environments, this just disables security locally to make testing/querying easier